src/components/activity/ActivityForm.svelte

```diff
@@ -99,6 +105,21 @@
     }
   }
 
+  /**
+   * Transforms computed attributes to conform to ParametersMap and ArgumentsMap
+   * so we can render computed attributs as form parameters.
+   */
+  function setFormParametersComputedAttributes(
+    schema: ValueSchema | undefined,
+    computedAttributes: ArgumentsMap | undefined,
```

Comment:
Does an ArgumentsMap allow for non-struct computed attributes?

Reply:
Yes sir